packages/dx-react-grid-material-ui/src/templates/table-group-cell/container.jsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { withStyles } from '@material-ui/core/styles';
+import { getStickyCellStyle } from '../utils';
+
+const styles = theme => ({
+  wrapper: {
+    ...getStickyCellStyle(theme),
+    display: 'inline-block',
+  },
+});
+
+const ContainerBase = ({
+  children, style, classes, className, ...restProps
+}) => (
+  <div
+    className={classNames(classes.wrapper, className)}
+    style={style}
```

Comment:
Should we use the following syntax instead of current?

```jsx
const ContainerBase = ({
  children, left, classes, className, ...restProps
}) => (
  <div
    className={classNames(classes.wrapper, className)}
    style={{ left, ...style }}
    ...
```